### ISSUE TITLE
TRANSFER LEARNING USING DLIB

### ISSUE THREAD

**Issue by lebaloc** [davisking/dlib#1476]
Hi,
Is it's possible to use dlib for transfer learning (fine-tuning)?  I want to adapt the layers from the resnet model and just change a few last convolution layers, the fc and loss layer. In addition, during the training process, i want to maintain the weight of the adapted layered of resnet model (not perform backpropagation in these layers).
Thanks in advance,
lebaloc

**Comment by davisking** [comment]
Yes, you can do all those things.  Please read the documentation, in particular, start with the two introductory example programs.  It's all documented.
